guide/sections/global-services.adoc

```diff
@@ -117,3 +117,12 @@ TODO: to be completed
 Procedures pertinent to the Global Monitor will be described here once validated through testing during the WIS2 pilot phase.
 
 TODO: to be completed
+
+=== Global Registry ===
+
+* The roles of the Global Registry is to make available the information required for Global Infrastructure and WIS2 nodes to connect with each other. 
```

Comment:
I don't think we need to introduce the term" Global Infrastructure". I'd prefer to keep Global Services. 
We also have to be consistent. Here WIS2 Node is used. In other PR it is WIS Node. So far we used WIS2 Nodes. That would be my preference.